var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -206,7 +219,9 @@ def define_feature(key, variant=None):
         if "+dbus" in spec:
             features.append("dbus_linked")
         if "+network" in spec:
-            features += ["openssl_linked", "openssl", "libproxy"]
+            features += ["openssl_linked", "openssl"]
+            if sys.platform == "linux":
+                features += ["libproxy"]
```

Comment:
```suggestion
            features.extend(["openssl_linked", "openssl"])
            if sys.platform == "linux":
                features.append("libproxy")
```

For readability we could flip these to an extend and an append call to make them more consistent with the previous features added.